### PR TITLE
Adding new Analytics content to lesson plan

### PR DESCRIPTION
**Pull request by emchap**
Doing the lesson plan first; if this passes muster I'll port to the LMS content.

### CODE REVIEW

File: trainings/admin-webinar/lesson-plan.md [hunk 1]
```diff
@@ -177,7 +177,18 @@ The admin section of the Sourcegraph site offers a variety of subsections that c
...
+### Viewing aggregate user activity
+
+To view aggregate user activity, go to the Users section in the Analytics section of the site admin panel. This shows you total active users, total registered users, and provisioned user licenses. The line graph at the top will show you user actions triggered over time or unique users using the app over time, toggled below the graph. You can also see DAU, WAU, and MAU at a glance, averaged out for the time frame you're viewing. Additionally, you'll see a bar graph showing how many days users used the application over the time period you specified—if you're seeing users mostly using Sourcegraph 1 or 2 days per time period, you may want to discuss how to reengage them with your CE.
+
+🔎 The trainer should show:
+
+* How to access the Users Analytics page
+* How to interpret the graph data
+
+❗️ The Usage Stats page has some of the same data; if asked, you can be honest with the user that the User Analytics page is a reimagining of the aggregate data in alignment with our overall value prop, but the Usage Stats page is still helpful to see individual activity.
```
Comment (ryphil) [inline trainings/admin-webinar/lesson-plan.md:189]: One other callout here - The usage-stats page is still the page which should be used when determining seat-license usage in context of billing. The legacy usage stats page will still be used for this until we move the new analytics out of 'experimental' stage.

Comment (emchap) [reply trainings/admin-webinar/lesson-plan.md:189]: In general we don't encourage customers to use in-app data for billing, at all (we discuss the bill with them using data in Looker). (That said the data they need is actually on the License or Overview page; Usage Stats doesn't show the current registered user count, which is how we bill.)

See [Slack](https://sourcegraph.slack.com/archives/C03CSAER9LK/p1658768307987479) for why this is our general stance. :)

Comment (emchap) [reply trainings/admin-webinar/lesson-plan.md:189]: @sourcegraph/customer-engineering Do you think this warrants its own callout for newer team members, or is our onboarding clear enough that we use ping data for this?

File: trainings/admin-webinar/lesson-plan.md [hunk 2]
```diff
@@ -276,8 +287,66 @@ User activity can be tracked in-app and via the API. This is the basis for user
...
+* How to access the Notebooks graph
+* How to customize the time saved data
+
+### Conclusion
+
+The Analytics functionality will allow your team to track time saved and show the value Sourcegraph brings to your team.
+
+### Resources
+
+* TBD once they are released
```
Comment (ryphil) [inline trainings/admin-webinar/lesson-plan.md:347]: We're iterating on [this page](https://docs.sourcegraph.com/admin/admin_analytics). Is this type of resources you're looking for?

Comment (emchap) [reply trainings/admin-webinar/lesson-plan.md:347]: Yes! Wasn't coming up in my initial Docs search. Will add.

File: trainings/admin-webinar/lesson-plan.md [hunk 3]
```diff
@@ -276,8 +287,66 @@ User activity can be tracked in-app and via the API. This is the basis for user
...
+Notebooks in Sourcegraph allow team members to create living documentation collections of searches, references to files, and explanations for why/why not to do something in the code. The Notebooks section of Analytics shows the number of notebooks created, the number of views of them, and the number of times an embedded block of live-updated content is "run" (updated). Similar to the other panels, you can track the amount of time this saves your team when onboarding a new teammate or sharing context.
+
+🔎 The trainer should demonstrate:
+
+* How to access the Notebooks graph
+* How to customize the time saved data
+
+### Conclusion
+
+The Analytics functionality will allow your team to track time saved and show the value Sourcegraph brings to your team.
```
Comment (ryphil) [inline trainings/admin-webinar/lesson-plan.md:343]: Does it make sense to include a CTA to provide feedback incase there are visualizations they would like to see?

**Review by ryphil — APPROVED** [review]
A few minor comments, but overall LGTM.

File: trainings/admin-webinar/lesson-plan.md (hunk 3 above)
Comment (emchap) [reply trainings/admin-webinar/lesson-plan.md:343]: I think "if the customer requests stuff, tell Product" is a general best practice so probably doesn't warrant a CTA, but @sourcegraph/customer-engineering , what do you think?

**Comment by emchap** [comment]
@ryphil Thanks for your review! Super helpful.

**Pull request merged**